pyairtable/models/_base.py

```diff
@@ -86,7 +95,8 @@ def deleted(self) -> bool:
 
     def __setattr__(self, name: str, value: Any) -> None:
         # Prevents implementers from changing values on readonly or non-writable fields.
-        if name in self.__class__.__fields__:
+        # Mypy can't tell that we are using pydantic v1.
+        if name in self.__class__.__fields__:  # type: ignore[operator, unused-ignore]
```

Comment:
I was able to drop the `unused-ignore` portion of this without reintroducing any failures.

Reply:
Are you sure you tried with v2? I think v2's BaseModel does not have __fields__, so mypy complains here.

Reply:
🤷  I'm running `tox -e mypy` and it's using the latest pydantic version, but I'm not sure it's worth our time to keep tweaking this particular nuance. Having `# type: ignore` feels kinda icky regardless, so who cares which options it needs? 😁

Reply:
Yeah, it needs ignore-unused for v1 because the warning only occurs with pydantic v2.